app/konnect/getting-started/migration.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Migrating from Kong Gateway On-Premises to Kong Konnect
+---
+
+The success of {{site.base_gateway}} can be attributed to its flexible deployment model, 
+exceptional performance, broad extensibility, and Open Source and Enterprise licensing models.
+This has resulted in a large deployment base over a variety of environments and topologies. 
+
+As organizations grow and scale they need for more advanced capabilities while lowering 
+operational complexity. Features such as strong [multi-tenancy](https://konghq.com/blog/enterprise/multi-tenancy), 
+federated API management, advanced security integrations, and others. With {{site.konnect_product_name}}, 
+users have access to a full featured API management platform which provides managed components and 
+builds on the lessons learned from deep customer usage of {{site.base_gateway}} and API management best practices. 
+
+This document will provide a guide for {{site.base_gateway}} users looking to migrate to {{site.konnect_product_name}}. 
+
+## Migration Guide
+
+The following will provide key details for completing a successful migration from {{site.base_gateway}} on-premises to 
+{{site.konnect_product_name}}. This document focuses on migrating a [Hybrid](/gateway/latest/production/deployment-topologies/hybrid-mode/) 
+or [Traditional](/gateway/latest/production/deployment-topologies/traditional/)
+deployment to Konnect, for other deployment modes see the following:
+
+* [{{site.kic_product_name}} (KIC)](/kubernetes-ingress-controller/latest/)
+    
+    If you run KIC on premises, migrating to Konnect is straightforward. 
+    The [Kong Ingress Controller for Kubernetes Assocation](/konnect/gateway-manager/kic/)
+    documentation provides details on linking your KIC deployment to a KIC based Control Plane
+    in {{site.konnect_product_name}}.
+
+* [DB-less mode](/gateway/latest/production/deployment-topologies/db-less-and-declarative-config/)
+
+    If you're running a DB-less deployment and interested in migrating to Konnect, Kong recommends migrating to 
+    Hybrid mode deployment to take advantage of the full capabilities of Kong Konnect. Please reach out to
+    a Kong representative for assistance with this migration.
+
```

Comment:
```suggestion
## Migration Guide

The following will provide key details for completing a successful migration from {{site.base_gateway}} on-premises to 
{{site.konnect_product_name}}. 

<Insert Flow-chart of path or steps to migration






This document focuses on migrating a [Hybrid](/gateway/latest/production/deployment-topologies/hybrid-mode/) 
or [Traditional](/gateway/latest/production/deployment-topologies/traditional/)
deployment to Konnect, for other deployment modes see the following:

* [{{site.kic_product_name}} (KIC)](/kubernetes-ingress-controller/latest/)
    
    If you run KIC on premises, migrating to Konnect is straightforward. 
    The [Kong Ingress Controller for Kubernetes Assocation](/konnect/gateway-manager/kic/)
    documentation provides details on linking your KIC deployment to a KIC based Control Plane
    in {{site.konnect_product_name}}.

* [DB-less mode](/gateway/latest/production/deployment-topologies/db-less-and-declarative-config/)

    If you're running a DB-less deployment and interested in migrating to Konnect, Kong recommends migrating to 
    Hybrid mode deployment to take advantage of the full capabilities of Kong Konnect. Please reach out to
    a Kong representative for assistance with this migration.

```

Reply:
Agree with diagram. Resolving this comment though, as I'll just add a diagram directly instead of suggesting a placeholder.